Gemfile

```diff
@@ -20,7 +21,7 @@ if RUBY_ENGINE == 'rbx'
   end
 end
 
-rails = ENV['RAILS'] || '~> 6.0.4'
+rails = ENV['RAILS'] || '~> 6.1.0'
```

Comment:
we could move up the `rails` initialization, before `sqlite` and use the var instead, but I can see how its equivalent. so maybe its OK as-is